<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ADS67" name="Test" tests="14" failures="5" timestamp="4 Oct 2017 12:58:21 GMT" time="1.701" errors="0">
  <testcase name="ArthaDB_ProdCodes_Check_TC001" time="0.029" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_AuditDates_Check_TC002" time="0.01" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Account_Number_Check_TC003" time="0.006" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ChargeType_Check_TC004" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Invoice_Number_Check_TC005" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Transaction_Amount_Check_TC006" time="0.015" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ChargeSign_Check_TC007" time="0.005" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ChargeSign_Check_TC007(Arthadb_Automation.java:343)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ChargeSign_Check_TC007 -->
  <testcase name="ArthaDB_Record_Type_Check_TC008" time="0.005" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J9160067066C122,J91606706e6C122, expected:&amp;lt;0&amp;gt; but was:&amp;lt;3&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J9160067066C122,J91606706e6C122, expected:<0> but was:<3>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Record_Type_Check_TC008(Arthadb_Automation.java:386)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Record_Type_Check_TC008 -->
  <testcase name="ArthaDB_Customer_FirstName_Check_TC009" time="0.01" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="CUSTOMERS-First Name check is Failed at SSN=620-74-4374, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: CUSTOMERS-First Name check is Failed at SSN=620-74-4374, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Customer_FirstName_Check_TC009(Arthadb_Automation.java:428)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Customer_FirstName_Check_TC009 -->
  <testcase name="ArthaDB_Customers_SSN_Check_TC010" time="0.011" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="CUSTOMERS- SSN check is Failed at CustomerPK=5, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: CUSTOMERS- SSN check is Failed at CustomerPK=5, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Customers_SSN_Check_TC010(Arthadb_Automation.java:470)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Customers_SSN_Check_TC010 -->
  <testcase name="ArthaDB_Record_Type_Check_TC011" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Premise_City_Check_TC012" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_PrimaryPhoneNumber_TC013" time="0.004" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ZipCode_TC013" time="0.015" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="Customer - PrimaryPhoneNumber check is Failed at SSN=563-13-3480,405-05-7405,348-37-6100, expected:&amp;lt;0&amp;gt; but was:&amp;lt;3&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: Customer - PrimaryPhoneNumber check is Failed at SSN=563-13-3480,405-05-7405,348-37-6100, expected:<0> but was:<3>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ZipCode_TC013(Arthadb_Automation.java:612)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ZipCode_TC013 -->
</testsuite> <!-- Test -->
